llvm/include/llvm/CodeGen/TargetLowering.h

```diff
@@ -268,6 +268,7 @@ class LLVM_ABI TargetLoweringBase {
     CmpArithIntrinsic, // Use a target-specific intrinsic for special compare
                        // operations; used by X86.
     Expand,            // Generic expansion in terms of other atomic operations.
+    CustomExpand,      // Custom target-specific expansion using TLI hooks.
```

Comment:
Does it make more sense to call it `Custom` instead of `CustomExpand`, in the light of the naming in SelectionDAG?